openmmtools/states.py

```diff
@@ -1750,11 +1750,11 @@ class SamplerState(object):
 
     Parameters
     ----------
-    positions : Nx3 simtk.unit.Quantity
+    positions : Nx3 simtk.unit.Quantity or list(simtk.openmm.Vec3)
```

Comment:
I'd revert changes this as it seems that we accept simple lists without units. You can maybe add in the description below that `Quantities` that use `List[Vec3]` values are supported, or actually any value that `np.array(input)` supports.

Reply:
I don't know what would be a good description.
positions : simtk.unit.Quantity of Nx3 or a lists of Vec3s. 
What do you think?

Reply:
The outer class of what is returned by `openmm_positions` is still a `Quantity` technically. It should be `Quantity(list(Vec3))` so I'd leave just `Quantity`. There are other types of `_value` that work (e.g., list of lists, tuple of lists, list of ndarray, etc.) so it'd be difficult to enumerate them all anyway.